samples/core/visualization/confusion_matrix_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .confusion_matrix import confusion_matrix_pipeline
+from ...test.util import run_pipeline_func, TestCase
+
+import kfp
+
+run_pipeline_func([TestCase(pipeline_func=confusion_matrix_pipeline,
```

Comment:
For better test coverage, it's important to write verification func to verify pipeline output result. See two step pipeline, or lightweight python component v2 sample tests as examples.

Our testing plan is that, one or two representative pipelines will have verification for full mlmd data.
The other pipelines should only verify expected output. E.g. in this case, it's valuable to verify the output ml-pipeline-ui-metadata artifact has the correct name and payload file content.

Reply:
Thank you for the reference! I added a simple verification for this case. And once we have all visualization sample, we can have one test for all visualization output contents.

Reply:
Thanks, looks enough for now!

The format looks a little different from what the yapf tool automatically formats, can you confirm?
https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#coding-style

Reply:
Thank you Yuan! I just applied yapf setting, I will apply formatting in coming PRs.